src/main/resources/application.properties

```diff
@@ -16,5 +16,5 @@ spring.mail.protocol=smtp
 mail.smtp.starttls.enable=true
 mail.debug=true
 mail.smtp.auth=true
-mail.activationURL=http://localhost:8080/dashboards/
+mail.activationURL=http://localhost:8080/confirmation?emailUUID=
```

Comment:
If it'll be possible, check how we can to replace
`http://localhost:8080` with some variable and use it with `interpolation`

Reply:
Okey, I will try

Reply:
It doesn't work. I will try another way to solve it.

Reply:
It should work